src/components/Modals/Send/Form.tsx

```diff
@@ -62,6 +62,7 @@ export const Form: React.FC<SendFormProps> = ({ asset: initialAsset, accountId }
 
   const handleAssetSelect = async (asset: Asset) => {
     methods.setValue(SendFormFields.Asset, { ...asset, ...marketData })
+    methods.setValue(SendFormFields.Input, '')
```

Comment:
On select asset, we want to clear the `Input` form state because it can be a different chain or whatever different than the previously selected one